pkg/kv/kvserver/allocator/mmaprototype/load.go

```diff
@@ -556,13 +539,23 @@ func loadSummaryForDimension(
 	)
 	if fractionAbove > meanFractionSlow {
 		summ = overloadSlow
+		reason = "load is >10% above mean"
 	} else if fractionAbove < meanFractionLow {
 		summ = loadLow
+		reason = "load is >10% below mean"
 	} else if fractionAbove >= meanFractionNoChange {
 		summ = loadNoChange
+		reason = "load is within 5-10% of mean"
 	} else {
 		summ = loadNormal
+		reason = "load is within 5% of mean"
 	}
+
+	defer func() {
+		log.KvDistribution.VEventf(ctx, 3,
+			"load summary for dim=%s (n%vs%v): %s, reason: %s [fractionUsed=%.2f%% meanUtil=%.2f%% capacity=%d load=%d]",
+			dim, nodeID, storeID, summary, reason, fractionUsed*100, meanUtil*100, capacity, load)
+	}()
```

Comment:
Is this adding too much noise to the logs or do we like it? we could also only log it when the summary is above some threshold. Open to other opinions/ideas as well.